### DISCUSSION TITLE
Super cool, merci

### DISCUSSION THREAD

**Discussion by flamme-demon** [X3n0m0rph59/eruption#85] [General]
Hi,

After an installation that I did not necessarily master. I persevered and I am very happy with the result.
So I wanted to thank you and I hope the GUI will be as good.

In any case thank you.

Model: Roccat Vulkan Pro

**Comment by Marethyu61** [comment]
Hi,

This little message to say that the eruption shortcut accessible from the desktop on Roccat Kone Aimo is really very practical for a layman like me. I thank you very much for your service!

**Reply by flamme-demon** [reply]
Hello,

If you want help to continue on this model it is with pleasure.
I have this template on Windows and Linux.
